### PR TITLE
Merge subtree raduom/hysterical screams into plutus-hysterical-screams

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

**Comment by zeme-iohk** [comment]
This PR simply pulls `hysterical-screams` into `plutus-apps/plutus-hysterical-screams` as per https://input-output.atlassian.net/browse/PLT-552
No other file has been modified.
Git history has been maintained. 
Then actual "integration" into the repo can begin

**Comment by raduom** [comment]
As funny as it sounds "plutus-hysterical-screams", I am thinking maybe we want to change the name? :P

**Comment by koslambrou** [comment]
@raduom I think you might be best person to find a representative name :D

**Comment by raduom** [comment]
Let's leave it as is then ^^

**Review by raduom — APPROVED** [review]
We can go with it as it is and have any changes that make sense done after we have the project.

**Review by raduom — APPROVED** [review]

File: plutus-hysterical-screams/.github/workflows/haskell-ci.yml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# This GitHub workflow config has been generated by a script via
```
[the remark this answers is not in the document]
Comment (zeme-iohk) [reply plutus-hysterical-screams/.github/workflows/haskell-ci.yml:1]: Im going to keep this just for my reference as I integrate it with the CI of plutus-apps. I will remove it in due time

File: plutus-hysterical-screams/LICENSE [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2022
```
[the remark this answers is not in the document]
Comment (zeme-iohk) [reply plutus-hysterical-screams/LICENSE:1]: Ok

**Pull request merged**